content/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -0,0 +1,106 @@
+                </p>
+                <p>कुबेरनेट्स क्लस्टर में दो प्रकार के संसाधन होते हैं:
+                    <ul>
+                        <li><b>कंट्रोल प्लेन</b> क्लस्टर का समन्वय करता है</li>
+                        <li><b>नोड्स</b> वे कर्मचारी हैं जो एप्लिकेशन चलाते हैं</li>
+                    </ul>
+                </p>
+            </div>
+
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>सारांश:</h3>
+                    <ul>
+                        <li>कुबेरनेट्स क्लस्टर</li>
+                        <li>मिनिक्यूब</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i>
+                        कुबेरनेट्स एक प्रोडक्शन-ग्रेड, ओपन-सोर्स प्लेटफॉर्म है जो कंप्यूटर क्लस्टर के भीतर और उसके भीतर एप्लिकेशन कंटेनरों के प्लेसमेंट (शेड्यूलिंग) और निष्पादन को व्यवस्थित करता है।
+                    </i></p>
+                </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2 style="color: #3771e3;">क्लस्टर आरेख</h2>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_01_cluster.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><b>कंट्रोल प्लेन क्लस्टर के प्रबंधन के लिए जिम्मेदार है।</b> कंट्रोल प्लेन आपके क्लस्टर में सभी गतिविधियों का समन्वय करता है, जैसे एप्लिकेशन शेड्यूल करना, एप्लिकेशन की वांछित स्थिति को बनाए रखना, एप्लिकेशन को स्केल करना और नए अपडेट रोल आउट करना। </p>
+                <p><b>नोड एक VM या एक भौतिक कंप्यूटर है जो Kubernetes क्लस्टर में एक वर्कर मशीन के रूप में कार्य करता है।</b> प्रत्येक नोड में एक Kubelet होता है, जो नोड के प्रबंधन और कुबेरनेट्स नियंत्रण के साथ संचार करने के लिए एक एजेंट है। विमान। नोड में कंटेनर संचालन को संभालने के लिए उपकरण भी होने चाहिए, जैसे कि कंटेनरड या डॉकर। उत्पादन ट्रैफ़िक को संभालने वाले कुबेरनेट्स क्लस्टर में कम से कम तीन नोड होने चाहिए।</p>
```

Comment:
We usually write “kubelet” all-lowercase. Also, it's OK to transcribe Kubernetes into dévanāgarī here.